widgetssdk/src/main/java/com/glia/widgets/entrywidget/EntryWidgetController.kt

```diff
@@ -96,6 +96,8 @@ internal class EntryWidgetController @JvmOverloads constructor(
         isViewWhiteLabel: Boolean = view.whiteLabel
     ): List<EntryWidgetContract.ItemType> {
         val items = when (engagementState) {
+            is State.PreQueuing -> prepareItemsBasedOnOngoingEngagement(engagementState.mediaType, unreadMessagesCount, hasOngoingSC)
+            is State.Queuing -> prepareItemsBasedOnOngoingEngagement(engagementState.mediaType, unreadMessagesCount, hasOngoingSC)
```

Comment:
FYI. Although both conditions call exactly the same code`prepareItemsBasedOnOngoingEngagement(engagementState.mediaType, unreadMessagesCount, hasOngoingSC)` , when I unite the conditions like this `is State.PreQueuing, is State.Queuing`, compiler doesn't recognize the `engagementState.mediaType` call because these states have different set of params.